### ISSUE TITLE
[SECURITY] Supplement documentations of best practice

### ISSUE THREAD

**Issue by vincentgoat** [kubeedge/kubeedge#4014]
**What would you like to be added/modified**:

- [ ] Complete recommended security configuration documents
- [ ] Complete the documentation for each component on the Kubeedge. IO page
- [ ] Fix `MqttPassword` and `Token` plaintext storage in file `edgecore.yaml` problems
- [ ] Analyze and fix `InsecureSkipVerify` config of http client
- [ ] Integrate SLSA Build Provenance Action (FYI: https://github.com/marketplace/actions/slsa-build-provenance-action)
- [ ] Sigstore integration of SLSA (FYI: https://www.sigstore.dev/)

**Why is this needed**:
To enhance the KubeEdge security.
/help

**Comment by jxs1211** [comment]
hi, I want to take the task of ` Analyze and fix InsecureSkipVerify config of http client `, please assign it to me

**Comment by vincentgoat** [comment]
> hi, I want to take the task of `Analyze and fix InsecureSkipVerify config of http client`, please assign it to me

Sure, thanks a ton!

**Comment by vincentgoat** [comment]
> Start EventBus/ServiceBus/MetaServer with TLS encrypted.

We've already encrypted MetaServer by https://github.com/kubeedge/kubeedge/pull/4104.
And there is still a need for help with EventBus and ServiceBus encryption.

**Comment by jxs1211** [comment]
> > hi, I want to take the task of `Analyze and fix InsecureSkipVerify config of http client`, please assign it to me
> 
> Sure, thanks a ton!

I want to make sure that the task is originate from https://github.com/kubeedge/kubeedge/issues/4001, so the need of the task is `Set InsecureSkipVerify to false in http request`, right?

**Comment by vincentgoat** [comment]
> > > hi, I want to take the task of `Analyze and fix InsecureSkipVerify config of http client`, please assign it to me
> > 
> > 
> > Sure, thanks a ton!
> 
> I want to make sure that the task is originate from #4001, so the need of the task is `Set InsecureSkipVerify to false in http request`, right?

Yes, we should confirm at least we have one way to use https client with `InsecureSkipVerify` set to false.

**Comment by jxs1211** [comment]
> > > > hi, I want to take the task of `Analyze and fix InsecureSkipVerify config of http client`, please assign it to me
> > > 
> > > 
> > > Sure, thanks a ton!
> > 
> > 
> > I want to make sure that the task is originate from #4001, so the need of the task is `Set InsecureSkipVerify to false in http request`, right?
> 
> Yes, we should confirm at least we have one way to use HTTPS clients with `InsecureSkipVerify` set to false. And make sure that the HTTPS connection should be normal after setting `InsecureSkipVerify` to false. FYI: Some work has been done by #4015.

@vincentgoat #4378 for #4001

**Comment by fengshunli** [comment]
>> Integrate [scorecard](https://github.com/ossf/scorecard) in CI, and display a Scorecard badge on the KubeEdge repository.

assigned to me

**Comment by vincentgoat** [comment]
> > > Integrate [scorecard](https://github.com/ossf/scorecard) in CI, and display a Scorecard badge on the KubeEdge repository.
> 
> assigned to me

Thanks.
